### ISSUE TITLE
not signed

### ISSUE THREAD

**Issue by portal-draakje** [bridge-core/editor#1144]
**Summary**
the bug is not signed, that makes me worry the file has been edited.

**To Reproduce**

1. Go to google
2. Click on https://bridge-core.app/guide/download/index.html
3. Scroll down to just download for windows
4. Observe result windows will warn you because it's not signed

**Observed behavior**
not signed file
**Expected behavior**
the file is not signed
**Screenshots / File Attachments**
![image](https://github.com/bridge-core/editor/assets/69325629/dec790c8-c3e1-4f38-ac0e-f0909b675d3f)

**Platform (please complete the following information):**

-   OS: [e.g. Windows 11]
-   App Version: [2]

**Additional context**
i think it needs to be signed to show it has not been edited.

**Comment by portal-draakje** [comment]
i did see i needed to fill default info, so i did follow the info.

**Comment by outercloudstudio** [comment]
Yep, this is something I want to look into. I believe it costs money though.

**Comment by portal-draakje** [comment]
i think there are free options, it's mostly to show humans can trust your app.
